quiche/src/recovery/cubic.rs

```diff
@@ -1254,9 +1273,6 @@ mod tests {
         // Exit from the recovery.
         now += rtt;
 
-        // To avoid rollback
-        r.lost_count += MIN_ROLLBACK_THRESHOLD;
```

Comment:
Same comment as above regarding what this removal means

Reply:
Same answer :)